buildSrc/src/main/java/org/opensearch/gradle/zipplugin/ZipPublishExtension.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.gradle.zipplugin;
+
+public class ZipPublishExtension {
```

Comment:
I would replace 'zip` terminology with 'PluginDistribution` since this is reflects what exactly we intend to publish and where is it applicable, fe `PublishPluginDistributionExtension`

Reply:
since i have organized as folder under `zippublish/` I guess it should be good, the entire effort of this plugin is to publish the zips, hence added with start `Zip` :)

Reply:
I believe the notion of zip is implementation detail, what we are developing is publication mechanism for plugin distribution

Reply:
I am with @reta and would call this `PluginDistributionExtension`